Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1176,7 +1176,8 @@ def audit_line(line, _lineno)
       problem "Use `assert_match` instead of `assert ...include?`"
     end
 
-    if line.include?('system "npm", "install"') && !line.include?("Language::Node") && formula.name !~ /^kibana(\d{2})?$/
+    if line.include?('system "npm", "install"') && !line.include?("Language::Node") &&
+      formula.name !~ /^kibana(\@\d+(\.\d+)?)?$/
```

Comment:
Last nit! Indent `formula.name` by a single character so it's aligned with `line.include?`